.circleci/config.yml

```diff
@@ -162,7 +181,7 @@ jobs:
             
 
 workflows:
-  open_pr:
+  open_new_pr:
```

Comment:
Is this name special, ie used by either github or circleci to run it on the PR creation event?
Or is it just a name, and it's run on all commits or something?

Reply:
OK so workflows run only on PR, not on all commits, so I'm wondering what will cause the `merge_pr_to_main` workflow to run. Do we also have a webhook to run workflows on PR merges not just commits to open PRs? If so, is there anything to stop _this_ workflow from also running when we merge a PR? I can see that the other one is only on main, but this one doesn't exclude main

Reply:
OK we'll see how it goes. I'm struggling to follow that logic entirely, but if you're confident I'm confident (and the negative impact of being wrong is minimal)